includes/gateways/stripe/includes/payment-methods/class-give-stripe-card.php

```diff
@@ -147,6 +147,12 @@ public function addCreditCardForm( $form_id, $args, $echo = true ) {
 		 */
 		public function process_payment( $donation_data ) {
 
+            /**
+             * Early return to prevent legacy processing of the Stripe Credit Card gateway.
+             * See src/PaymentGateways/Gateways/Stripe/CreditCardGateway.php
+             */
+            return;
```

Comment:
@kjohnson I think it is not a request because you already handle this in the main class `Give_Stripe_Gateway`.  This means legacy payment processing logic will not run if the payment method is registered with a new gateway API.

Can we remove this?

Reply:
Yeah, I think Jon made a related change elsewhere, but this hadn't been removed.

Reply:
@ravinderk resolved by 19ec5f6.